recipes/msgpack-cxx/all/conanfile.py

```diff
@@ -14,6 +14,15 @@ class MsgpackCXXConan(ConanFile):
     topics = ("msgpack", "message-pack", "serialization")
     license = "BSL-1.0"
     no_copy_source = True
+    
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "use_boost": [True, False]
```

Comment:
usually I prefer `with_` but as the project uses `use_` I'm fine

Reply:
If it becomes a problem later it shouldn't be hard to change